pkg/rtc/rtc_linux_test.go

```diff
@@ -0,0 +1,115 @@
+// Copyright 2021 the u-root Authors. All rights reserved
+// Use of this source code is governed by a BSD-style
+// license that can be found in the LICENSE file.
+
+package rtc
+
+import (
+	"errors"
+	"io/ioutil"
+	"os"
+	"testing"
+	"time"
+
+	"golang.org/x/sys/unix"
+)
+
+func TestSet(t *testing.T) {
+	for _, tt := range []struct {
+		name string
+	}{
+		{
+			name: "success",
+		},
+		{
+			name: "error",
+		},
+	} {
+		t.Run(tt.name, func(t *testing.T) {
+			f, err := ioutil.TempFile("", "rtc-*")
+			if err != nil {
+				t.Error(err)
+			}
+			devs = []string{f.Name()}
+			f.Close()
+			rtc, err := OpenRTC()
+			if err != nil {
+				t.Error(err)
+			}
```

Comment:
not strictly needed but it is common to just put a 
defer rtc.Close()
at a place like this.

Reply:
I wasn't sure if the defer would have run on the end of t.Run() or the end of the loop where all the t.Run() are executed in so I went for the safe way and did a rtc.Close() a few lines further down where I would be sure I don't need the rtc anymore.

Reply:
ah, ok. I'd suggest putting a comment in about this, so that in three years someone else is as confused as I am :-)